src/components/mobile/molecules/GameTagModal/GameTagModal.tsx

```
@@ -62,29 +119,54 @@ const GameTagModal = ({
         </div>
-        <div css={S.tagWrapper}>
+        <div css={S.tagBottomWrapper}>
           <div css={S.textBox}>
             <span css={S.tagTextStyling}>서브태그</span>
+            <span css={S.subTagTextStyling}>(최대 3개)</span>
+          </div>
+          <div css={S.subTagWrapper(currentSubTag.length === 3)}>
+            {currentSubTag.map((tag, idx) => (
+              <div css={S.subTagChipStyling}>
+                <span>#{tag}</span>
+                <button
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)